components/src/themed/ThemedBottomNavigation.tsx

```diff
@@ -25,7 +25,8 @@ export const ThemedBottomNavigation: React.FC<ThemedBottomNavigationProps> = (pr
         props.activeColor ||
         (fullTheme.dark ? fullTheme.colors.primaryPalette.main : fullTheme.colors.textPalette.onPrimary.main);
     const inactiveColor =
-        props.inactiveColor || (fullTheme.dark ? fullTheme.colors.placeholder : fullTheme.colors.textPalette.highlight);
+        props.inactiveColor ||
+        (fullTheme.dark ? fullTheme.colors.placeholder : fullTheme.colors.overrides.bottomNavigation?.inactive);
```

Comment:
The override color should be used (if specified) regardless of whether the theme is light or dark. This should be something along the lines of:

```
props.inactiveColor || fullTheme.colors.overrides.bottomNavigation?.inactive || fullTheme.colors.placeholder;
```